README.md

```diff
@@ -30,20 +65,246 @@ public function registerBundles()
 ### Step 3: Import bundle routing file
 
 ```yaml
-# app/config/routing.yml
+# app/config/routing.yml or config/routing.yaml
 
-cr.dataflow:
+_cr.dataflow:
     resource: '@CodeRhapsodieEzDataflowBundle/Resources/config/routing.yaml'
 ```
 
+### Step 4: Update the database schema
+
+For this step, refer you to the [Code-Rhapsodie Dataflow Bundle installation guide](https://github.com/code-rhapsodie/dataflow-bundle#update-the-database).
```

Comment:
Please refer to the [Code-Rhapsodie Dataflow Bundle installation guide](https://github.com/code-rhapsodie/dataflow-bundle#update-the-database).